pyqube/rest/tests/test_generate_ticket.py

```diff
@@ -37,6 +37,7 @@ def setUp(self):
             "state": 1,
             "generated_by_ticket_kiosk": None,
             "generated_by_profile": None,
+            "generated_by_totem": None,
```

Comment:
This is not available yet :warning:

Reply:
Humm, I'm doing tests with totem branch. Maybe it's better use dev for now...
Now, I'm wondering if GraphQL works in dev branch due to schedules. Some time ago, in [discord](https://discord.com/channels/1087348768837488701/1087358955463917678/1311664115445665814) I asked about schedules of Queues and the solution was in totem branch. This code is already in dev branch?